src/main/java/Duke.java

```diff
@@ -1,10 +1,200 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.*;
+import java.io.IOException;
+
 public class Duke {
+    private static final String line = "____________________________________________________________\n";
+    public static File filePath = new File("data/duke.txt");
+    public static ArrayList<Task> tasks = new ArrayList<>();
+    public static int index = 0;
+
+
+    private static void appendToFile(String textToAppend) throws IOException {
+        FileWriter fw = new FileWriter("data/duke.txt", true); // create a FileWriter in append mode
+        fw.write(textToAppend);
+        fw.close();
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+
+        try {
+            printFileContents(filePath);
+        } catch (FileNotFoundException e) {
+            System.out.println("no file found");
+            File dataDirectory = new File("data");
+            dataDirectory.mkdir();
+            File dukeFile = new File("data/duke.txt");
+            //dukeFile.createNewFile();
+
+        }
+
+
+        String greet = "____________________________________________________________\n" +
+                " Hello! I'm Duke\n" +
+                "____________________________________________________________";
+        String exit = "____________________________________________________________\n" +
+                "Bye. Hope to see you again!\n" +
+                "____________________________________________________________\n";
+        System.out.println(logo);
+        System.out.println(greet);
+        System.out.print(" What can I do for you?\n");
+        System.out.print(line);
+
+        String input;
+        Scanner userInput = new Scanner(System.in);
+        input = userInput.nextLine();
+
+        while (!input.equals("bye")) {
+
+            if (input.contains("todo")) {
+                int dividerPosition = input.indexOf(" ");
+                String t = input.substring(dividerPosition + 1);
+                if(dividerPosition==-1 || t.equals("") || !input.contains(" ")){
+                    System.out.print(line);
+                    System.out.print("I don't quite understand." +  "\n" + "Please enter the command again." + "\n");
+                    System.out.print(line);
```

Comment:
Instead of repeatedly printing out a line, context, line consider making a helper function that prints the lines to make it cleaner

Currently:
```
System.out.print(line);
System.out.print(text);
System.out.print(line);
```

Proposed Changes:
```
private static void printTextWithLines(String text) {
    System.out.print(line);
    System.out.print(text);
    System.out.print(line);
}
```

`printTextWithLines("I don't quite understand." +  "\n" + "Please enter the command again." + "\n");`